File: samples_generated_code/samples_1/phi4/baseline_code_py/code_row_81.py
import sys

def getValueFromArray(arr, size, index):
    if 0 <= index < size:
        return arr[index]
    else:
        raise IndexError("Index out of range")

def main():
    id_sequence = [10, 20, 30, 40, 50]  # Example array
    try:
        idx = int(input().strip())
        result = getValueFromArray(id_sequence, len(id_sequence), idx)
        sys.exit(result)
    except ValueError:
        print("Invalid input: Please enter an integer.")
        sys.exit(1)
    except IndexError as e:
        print(e)
        sys.exit(2)

if __name__ == "__main__":
    main()

File: samples_generated_code/sample_2/phi4/baseline_code_c/code_row_81.c
#include <stdio.h>

int getValueFromArray(int arr[], int size, int index) {
    if (index >= 0 && index < size) {
        return arr[index];
    }
    return -1; // Return -1 for invalid indices
}

int main() {
    int id_sequence[] = {10, 20, 30, 40, 50};
    int size = sizeof(id_sequence) / sizeof(id_sequence[0]);
    
    printf("Enter an index: ");
    int index;
    scanf("%d", &index);
    
    int result = getValueFromArray(id_sequence, size, index);
    return result;
}